handbook/engineering/web/frontend-platform/wildcard-component-library.md

```diff
@@ -0,0 +1,66 @@
+# Wildcard Component Library (WCL)
+
+Wildcard Component Library is a comprehensive collection of reusable, design-approved components that can be composed to quickly build out different user journeys across the web app. WCL is stored in its own [directory](https://github.com/sourcegraph/sourcegraph/tree/main/client/wildcard), owned by the [Fronted Platform Team](https://about.sourcegraph.com/handbook/engineering/web/frontend-platform) and contributed to by all Sourcegraphers. WCL is a part of the Wildcard Design System.
+
+## WCL x Frontend Platfrom Team
+- The Frontend Platform Team is the owner of the design system and Wildcard Component Library components and needs to be up-to-date with all the changes and needs of other teams
+- The Frontend Platform Team should enable other teams make changes to the Wildcard Component Library by keeping them up-to-date (see Sharing updates to the components system), helping them plan their work, providing expertise, and reviewing the changes
+- The Fronted Platform Team **shouldn’t be responsible for implementing components for other teams**. Otherwise, we would create a bottleneck and FE Platform Team wouldn’t have time to focus on their team goals.
+- Other product teams should feel empowered to make changes and create components by themselves
+
+## Changing and creating components
+### Process of changing or creating a component by a product team
+What steps to take when one of the teams needs to update or add a Standard UI Component Library?
+
+1. The lead designer for the project creates a separate page in the project Figma file with full specification regarding the component (separate from the general project page or general design system files)
+2. The designer initiates the communication between the engineers from the product team and the Frontend Platform Team by marking them as reviewers in the Figma file containing the component specs
+3. Engineers from both teams review the document and plan the implementation work. Both teams can choose engineering leads for the project who will be responsible for communication and progress (higher efficiency and clear ownership).
```

Comment:
It would be great to mention here an issue template that @umpox added recently. Something like:

- Engineering lead should create an a proposal for a new component on the [FE Platform Project board](https://github.com/orgs/sourcegraph/projects/144) using [Wildcard proposal template](https://github.com/sourcegraph/sourcegraph/blob/main/.github/ISSUE_TEMPLATE/wildcard_proposal.md).